its/src/test/java/com/sonar/it/vbnet/WebConfigTest.java

```diff
@@ -39,17 +39,14 @@ public void init() {
   }
 
   @Test
-  @Ignore("Ignore until https://github.com/SonarSource/sonar-dotnet/issues/4111 gets fixed")
   public void should_raise_hotspot_on_web_config() throws Exception {
     final String projectName = "WebConfig.Vb";
```

Comment:
@pavel-mikula-sonarsource should this be updated to `WebConfig.VB`?

Reply:
Yes, it can be aligned. Will do